src/pages/minsuKang/component/TopNav.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+
+function TopNav(){
+    return(                    
+    <div className="TopNav_wrapper">
+        <h1 className="logo">WeBucks</h1>
+        <div className="nav">
+            <ul className="coffee">COFFEE</ul>
+            <ul className="menu">MENU</ul>
+            <ul className="store">STORE</ul>
+            <ul className="what">WHAT'S NEW</ul>   
```

Comment:
```suggestion
  <ul>
    <li className="coffee">COFFEE</li>
    <li className="menu">MENU</li>
    <li className="store">STORE</li>
    <li className="what">WHAT'S NEW</li>   
  </ul>
```

이 부분은 이렇게 바꾸는게 좋을 것 같습니다!

Reply:
아 네네 감사합니다 그러면 ul은 무조건 li를 써야지, 단독으로는 못 쓰는 군요?